frontend/src/metabase/common/components/CollectionPath/CollectionPath.styled.tsx

```diff
@@ -9,26 +10,31 @@ import { Ellipsis } from "./Ellipsis";
 /** When a cell is narrower than this width, breadcrumbs within it change significantly */
 const breadcrumbBreakpoint = "10rem";
 
-export const Breadcrumb = styled(ResponsiveChild)<{
+export const Breadcrumb = styled(
+  ResponsiveChild,
+  doNotForwardProps("maxWidth", "isSoleBreadcrumb", "index"),
+)<{
   maxWidth: string;
   isSoleBreadcrumb: boolean;
   index: number;
 }>`
-  ${({ maxWidth }) => {
-    return maxWidth ? `td & { max-width: ${maxWidth} };` : "";
+  ${({ maxWidth, isSoleBreadcrumb }) => {
+    return maxWidth
+      ? `td & { max-width: ${maxWidth} };`
+      : `max-width: calc(95cqw - ${isSoleBreadcrumb ? 1 : 3}rem);`;
```

Comment:
these magic numbers are throwing some ⛳ ⛳  for me

Reply:
Can you elaborate? Is this about 95cqw?

Reply:
we got a 95, a 1 and a 3. I believe they work now, but this sort of thing is hard to understand and hard to maintain.